apps/backend/src/modules/reviews/review.service.ts

```diff
@@ -27,7 +32,11 @@ class ReviewService implements Service {
 	}
 
 	public async create(
-		payload: ReviewCreatePayload,
+		payload: Pick<
+			UserAuthResponseDto,
+			"avatarUrl" | "firstName" | "id" | "lastName"
+		> &
+			ReviewCreatePayload,
```

Comment:
We need to pass it as a user object, just with limited properties
```suggestion
		payload:  {
		   user: Pick<
			UserAuthResponseDto,
			"avatarUrl" | "firstName" | "id" | "lastName"
		>} &
			ReviewCreatePayload,
```